packages/app/src/systems/Core/components/Mnemonic/Mnemonic.tsx

```diff
@@ -152,7 +161,7 @@ export function Mnemonic({
             variant="ghost"
             color="gray"
             leftIcon={<Icon icon="ClipboardText" color="gray8" />}
-            onPress={handlePast}
+            onPress={handlePasteButton}
```

Comment:
maybe change to handlePastePress ... not sure how the naming conventions work in this repo

Reply:
yeah, `handlePastePress` seems more appropriate.